pages/browser-support.html

```diff
@@ -4,37 +4,27 @@
 
 <h2>Current Active Support</h2>
 
-<table id="support-matrix">
-	<thead>
-		<tr>
-			<th></th>
-			<th>Internet Explorer</th>
-			<th>Edge, Opera, Yandex.Browser</th>
-			<th>Chrome, Firefox, Safari</th>
-			<th>iOS</th>
-			<th>Android</th>
-		</tr>
-	</thead>
-	<tbody>
-		<tr>
-			<th>jQuery</th>
-			<td>9+</td>
-			<td rowspan="2">Current</td>
-			<td rowspan="2">(Current - 1) and Current</td>
-			<td rowspan="2">7+</td>
-			<td rowspan="2">2.3, 4.0+</td>
-		</tr>
-		<tr>
-			<th>jQuery Compat</th>
-			<td>8+</td>
-		</tr>
-	</tbody>
-</table>
+<ul>
+	<li>Internet Explorer: 9+</li>
+	<li>Chrome: (Current - 1) and Current</li>
+	<li>Edge: Current</li>
+	<li>Firefox: (Current - 1) and Current</li>
+	<li>Safari: (Current - 1) and Current</li>
+	<li>Android: 4.0+ (plus limited 2.3 support)</li>
+	<li>iOS: 7+</li>
+	<li>Opera: Current</li>
+	<li>Yandex.Browser: Current</li>
+</ul>
```

Comment:
It may be worth introducing Desktop & Mobile headings like https://github.com/jquery/sizzle/wiki#user-content--browser-support , which would allow for alphabetization within each category.